### ISSUE TITLE
 fix cost function bug.

### ISSUE THREAD

**Issue by pshrohit** [pshrohit/Tic-Tac-Toe#1]
In prediction  and  y labels we have size of matrix [1,2] and [1]. I think we need one hot parameters for labels.

`tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits_v2(logits=prediction,labels= y))`

![screenshot 212](https://user-images.githubusercontent.com/34619517/48430955-878bd900-e796-11e8-85a2-099d53137956.png)
